cmd/imagestream.go

```diff
@@ -63,17 +67,30 @@ func (o *ImageStreamCleanupOptions) cleanupImageStreamTags(cmd *cobra.Command, a
 		return
 	}
 
-	commitHashes, err := git.GetCommitHashes(o.RepoPath, o.CommitLimit)
-	if err != nil {
-		log.WithError(err).WithField("RepoPath", o.RepoPath).WithField("CommitLimit", o.CommitLimit).Fatal("Retrieving commit hashes failed.")
+	var commits []string
+	if o.Tag {
+		if !git.IsValidSortValue(o.Sorted) {
+			log.WithField("namespace", o.Namespace).Fatal("Invalid sort flag")
```

Comment:
why is the namespace relevant when providing an invalid sort flag? Shouldn't the user input be validated before doing any potentially destructive business logic?